src/helpers/manage-merge-queue.ts

```diff
@@ -95,32 +96,34 @@ export const removePrFromQueue = async (pullRequest: PullRequest) => {
   }
 };
 
-const addPrToQueue = async (pullRequest: PullRequest, queuePosition: number) =>
+const addPrToQueue = async (pullRequest: PullRequest, queuePosition: number, skip_auto_merge: string | undefined) =>
   Promise.all([
     octokit.issues.addLabels({
       labels: [`${QUEUED_FOR_MERGE_PREFIX} #${queuePosition}`],
       issue_number: context.issue.number,
       ...context.repo
     }),
-    enableAutoMerge(pullRequest.node_id)
+    enableAutoMerge(pullRequest.node_id, skip_auto_merge)
```

Comment:
Let's remove this `Promise.all` and put the if statement around the `enableAutoMerge` method so we don't need to change its signature

Reply:
```suggestion
    const addPrToQueue = async (pullRequest: PullRequest, queuePosition: number, skip_auto_merge?: string) => {
      await octokit.issues.addLabels({
        labels: [`${QUEUED_FOR_MERGE_PREFIX} #${queuePosition}`],
        issue_number: context.issue.number,
        ...context.repo
      });
      if (Boolean(skip_auto_merge)) {
        core.info('Skipping auto merge per configuration.');
        return;
      }
      await enableAutoMerge(pullRequest.node_id);
    }
```